<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fAuditorySettings</class>
 <widget class="QWidget" name="fAuditorySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_log">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>391</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Log</string>
   </property>
   <widget class="QTextEdit" name="textEdit_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_auditories">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Auditories</string>
   </property>
   <widget class="QListWidget" name="listWidget_auditories">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_datetime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>DateTime Settings</string>
   </property>
   <widget class="QPushButton" name="pushButton_setDateTime">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set DateTime</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>HH:mm  dd.MM.yy </string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_sequritysettings">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>391</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Sequrity Settings</string>
   </property>
   <widget class="QPushButton" name="pushButton_setWorkers">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Workers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_getWorkers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Workers</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_allWorkers">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragOnly</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_WorkersInAud">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DropOnly</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
